src/wp-admin/setup-config.php

```diff
@@ -449,15 +447,17 @@ function setup_config_display_header( $body_classes = array() ) {
 	// Add core ui class.
 	$body_classes[] = 'cp-installation';
 	$body_classes[] = 'wp-core-ui';
+	$dir_attr       = '';
 	// Add rtl class, if needed.
 	if ( is_rtl() ) {
 		$body_classes[] = 'rtl';
+		$dir_attr      = ' dir="rtl" ';
 	}
 	// Set the content type.
 	header( 'Content-Type: text/html; charset=utf-8' );
 	// Print out the page header.
 	echo '<!DOCTYPE html>' . "\n";
-	echo '<html xmlns="http://www.w3.org/1999/xhtml" ' . get_language_attributes() . '>' . "\n";
+	echo '<html xmlns="http://www.w3.org/1999/xhtml"' . $dir_attr . get_language_attributes() . '>' . "\n";
```

Comment:
The three changes with `$dir_attr` seem unrelated to this PR. 
The space between the `"` and `'` is still needed.

Reply:
This came in as part of:
https://core.trac.wordpress.org/changeset/42217

Will need to review if this is needed.

Also the $dir_attr content is padded with spaces so it's find to remove the space between the quotes if this stays.

Reply:
There is a case where $dir_attr is empty (all LTR sites), so the space is needed for the language attribute to be separate.

Reply:
Discussed at PR review